typing/typemod.ml

```diff
@@ -1647,26 +1761,52 @@ and type_structure ?(toplevel = false) funct_body anchor env sstr scope =
   let rec type_struct env sstr =
     Ctype.init_def(Ident.current_time());
     match sstr with
-    | [] -> ([], [], env)
-    | pstr :: srem ->
+    | [] -> ([], [], [], env)
+    | pstr :: srem -> begin
```

Comment:
Same stylistic comment.  Please push begin...end to the "tail expression" (here it's not even required).

Reply:
done